src/components/Home.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AddNewBook from './AddNewBook';
+
+function Home() {
```

Comment:
- [ ] Let's use ES6 syntax for our function. Kindly change this to an arrow function

Reply:
this is clearly your preference and not part of the requirements.